fetch.js

```diff
@@ -558,13 +558,13 @@ export function fetch(input, init) {
 
     xhr.onerror = function() {
       setTimeout(function() {
-        reject(new TypeError('Network request failed'))
+        reject(new TypeError('Network request ('+request.method+' '+request.url+') failed'))
```

Comment:
These would be neater as string template literals, but since those were not currently in use in the codebase, I opted to match the existing style of plain strings.